src/actions/_thunkAction.js

```diff
@@ -1,16 +1,20 @@
 const thunkAction = actionFn => (...args) => {
   const action = actionFn(...args);
-  return dispatch => (
-    new Promise((resolve, reject) => (
+  return dispatch => {
+    const promise = new Promise((resolve, reject) => (
       dispatch(action).then(resolvedAction => {
+
         if (resolvedAction.error) {
           reject(resolvedAction);
           return;
         }
         resolve(resolvedAction);
       }, reject)
     ))
-  );
+
+    promise.catch(console.log);
```

Comment:
This is probably ok for DEBUG mode but for PRODUCTION it would be better to just catch and do nothing. We should think about adding rio debug mode.